# Request
string question
---
# Response
bool success
string message
Plan plan
